src/Template/SentencesLists/show.ctp

```diff
@@ -154,7 +154,9 @@ $this->set('title_for_layout', $this->Pages->formatTitle($listName));
     );
 
     if ($permissions['canAddSentences']) {
-        echo $this->Html->div('edit-list-name', $editImage);
+        if($permissions['canEdit']){
+            echo $this->Html->div('edit-list-name', $editImage);
+        }

```

Comment:
I think all of this should probably happen outside the `canAddSentences` check. As is, setting "Who can add/remove sentences" to "No one (list inactive)" also disables changing the list name, which seems weird.

Also, maybe instead of checking the `canEdit` permission twice, only check whether `$editImage` is defined or not. Then we have less coupling and it's easier to see that we won't attempt to show an undefined image.

Reply:
Seems when edit permissions are set to `no_one` the edit icon will be displayed but attempts at editing the name fail possibly due to the following permission checks
https://github.com/Tatoeba/tatoeba2/blob/03d2d6fc04074d076c33fa6865f89acf753e48e0/src/Model/Table/SentencesListsTable.php#L631-L640
https://github.com/Tatoeba/tatoeba2/blob/03d2d6fc04074d076c33fa6865f89acf753e48e0/src/Model/Entity/SentencesList.php#L30-L34

This made me believe `no_one` should be able to edit the name after a list in "No one (list inactive)". 

I am not 100% sure how the `isEditableBy` check is utilised throughout the rest of the project, so my solution had simply nested the permissions. I was trying not to touch the  controllers, entities or modelTables over what I think is a cosmetic QOL issue.

Reply:
Okay, then I'll just merge this as-is. Thanks!